streams/shared/src/main/scala/zio/stream/ZTransducer.scala

```diff
@@ -539,7 +539,7 @@ object ZTransducer {
   /**
    * Creates a transducer that returns the first element of a chunk, if it exists.
    */
-  def headOption[O]: ZTransducer[Any, Nothing, O, Option[O]] =
+  def head[O]: ZTransducer[Any, Nothing, O, Option[O]] =
```

Comment:
Another thing I realized is that this transducer will emit the very first element of the stream; not the first element of every chunk.

Same for `last`.

Reply:
These are useful transducers so let's just fix the scaladoc to reflect what they do.

Reply:
Ah, yeah, that's true. Not sure why I messed that up when even one of my debounce tests was explicitly checking for that. 😅

I updated the wording.